packages/material-ui/src/ArrayFieldItemTemplate/ArrayFieldItemTemplate.tsx

```diff
@@ -38,55 +38,57 @@ export default function ArrayFieldItemTemplate<
     minWidth: 0,
   };
   return (
-    <Grid container={true} alignItems='center'>
-      <Grid item={true} xs style={{ overflow: 'auto' }}>
-        <Box mb={2}>
-          <Paper elevation={2}>
-            <Box p={2}>{children}</Box>
-          </Paper>
-        </Box>
-      </Grid>
-
-      {hasToolbar && (
-        <Grid item={true}>
-          {(hasMoveUp || hasMoveDown) && (
-            <MoveUpButton
-              style={btnStyle}
-              disabled={disabled || readonly || !hasMoveUp}
-              onClick={onReorderClick(index, index - 1)}
-              uiSchema={uiSchema}
-              registry={registry}
-            />
-          )}
-          {(hasMoveUp || hasMoveDown) && (
-            <MoveDownButton
-              style={btnStyle}
-              disabled={disabled || readonly || !hasMoveDown}
-              onClick={onReorderClick(index, index + 1)}
-              uiSchema={uiSchema}
-              registry={registry}
-            />
-          )}
-          {hasCopy && (
-            <CopyButton
-              style={btnStyle}
-              disabled={disabled || readonly}
-              onClick={onCopyIndexClick(index)}
-              uiSchema={uiSchema}
-              registry={registry}
-            />
-          )}
-          {hasRemove && (
-            <RemoveButton
-              style={btnStyle}
-              disabled={disabled || readonly}
-              onClick={onDropIndexClick(index)}
-              uiSchema={uiSchema}
-              registry={registry}
-            />
-          )}
+    <Grid item={true} xs={12}>
```

Comment:
What about entirely removing the Grid `container` on [line 63](https://github.com/rjsf-team/react-jsonschema-form/blob/6ffc2ceb1f24e2661d0acbc0c3e09e5e443e9f27/packages/material-ui/src/ArrayFieldTemplate/ArrayFieldTemplate.tsx#L63) of `ArrayFieldTemplate` instead?

Reply:
Yes that makes more sense. There's just the question about what to do with the `key` prop on that Grid container. I could leave there and change the `Grid` to a `Box` or just remove it entirely along with the `key` prop. I'm not sure the `key` prop is necessary so maybe the latter makes more sense.

Reply:
Honestly the `key` prop is only required/used by React when it is being generated from some sort of map function, which this is not so you can remove it.